advanced_samples/Bulk_Upload/DotNet/BulkUploaderTest/BulkUploaderTest.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I think the 'coverlet' and 'Microsoft.NET.Test.Sdk' references can be removed. Also need to add StyleCop and FxCop analyzers to this project. 
Might also want to make sure everything's up to date but in my experience 'xunit.runner.visualstudio' 2.4.2 does not work.